tests/test_schemas.py

```diff
@@ -951,3 +953,36 @@ def custom_action(self, request, pk):
 
     assert inspector.should_include_endpoint(path, callback)
     assert inspector.get_allowed_methods(callback) == ["GET"]
+
+
+@unittest.skipUnless(coreapi, 'coreapi is not installed')
+@pytest.mark.parametrize(
+    ('method', 'expected'), [
+        ('get', True),
+        ('put', True),
+        ('patch', True),
+        ('delete', True),
+        ('GET', True),
+        ('PUT', True),
+        ('PATCH', True),
+        ('DELETE', True),
+        ('foo', False),
+        ('FOO', False),
+    ]
+)
+@modify_settings(INSTALLED_APPS={'append': 'django_filters'})
+def test_auto_schema_allows_filters(method, expected):
+    class AFilter(django_filters.FilterSet):
```

Comment:
Minor nitpick A* and An* prefixes don't help the understandability of the test objects. I'd probably just use `Filter` and `APIView` instead.

Reply:
I followed the pattern from that test file. There are classes like `AViewSet` and `AnAPIView` in the tests above.